docs/doxygen/mainpages/const_cpp.h

```diff
@@ -43,6 +43,9 @@ while <tt>\__WINDOWS__</tt> should be used for the platform tests.}
 @itemdef{\__WXOSX_IPHONE__, iPhone (UIKit)}
 @itemdef{\__WXOSX_COCOA__, macOS using Cocoa (AppKit)}
 @itemdef{\__WXOSX_MAC__, macOS (Cocoa)}
+@itemdef{\__WXDARWIN__, wxBase built on macOS or iPhone, synonym of <tt>\__DARWIN__</tt> (see below)}
+@itemdef{\__WXDARWIN_OSX__, wxBase built on macOS (can be used for with Apple and non-Apple GUIs)}
+@itemdef{\__WXDARWIN_IPHONE__, wxBase built on iPhone}
```

Comment:
```suggestion
@itemdef{\__WXDARWIN__, wxBase built on macOS or iPhone, synonym of <tt>\__DARWIN__</tt> (see below). This constant is available since wxWidgets 3.3.2.}
@itemdef{\__WXDARWIN_OSX__, wxBase built on macOS (can be used for with Apple and non-Apple GUIs). This constant is available since wxWidgets 3.3.2.}
@itemdef{\__WXDARWIN_IPHONE__, wxBase built on iPhone. This constant is available since wxWidgets 3.3.2.}
```